doozerlib/distgit.py

```diff
@@ -1952,6 +1952,9 @@ def _update_csv(self, version, release):
         csv_file, image_refs = self._get_csv_file_and_refs(csv_config)
         registry = csv_config['registry'].rstrip("/")
         image_map = csv_config.get('image-map', {})
+        namespace = csv_config.get('csv_namespace', self.runtime.group_config.get('csv_namespace', None))
```

Comment:
This doesn't seem to be correct. Here `namespace` will become be value for the operator that is being rebased. It should be relevant to the image that is referenced. So, it should be moved to the `for ref in image_refs` loop.

Reply:
Then namespace can be obtained from something like `meta.get("update-csv", {}).get("csv_namespace", group_csv_namespace)`